src/index.js

```diff
@@ -242,18 +252,19 @@ export default class ImageTool {
 
         this.uploadUrl(image.src);
         break;
-
-      case 'pattern':
+      }
+      case 'pattern': {
```

Comment:
These brackets make sense if logic in the case is complex. For now, it looks like extra useless code manipulation when you could just write `this.upload(event.data.data)` - nothing will change, because the code below doesn't have any conditions

Reply:
they are used to support `const` and `let` block-scoping

https://eslint.org/docs/rules/no-case-declarations

Reply:
they, indeed, but it doesn't make this recommendation important )

Reply:
and the variables used for better understanding of how `onPaste` works